recipes/openms/activate.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+export OPENMS_DATA_PATH=${CONDA_PREFIX}/share/OpenMS/
```

Comment:
I strongly suggest you find an other way to do this. This kind of environment variable is very hard to keep consistent especially when nesting environments

Reply:
Can you please print me to exactly where in your codebase this is used. I generally feel like there is often a better way and I would like to help avoid this activate script. It is often unmaintainable

Reply:
https://github.com/OpenMS/OpenMS/blob/947b5b966299300049cb6b9f9239b2462a244480/src/openms/source/SYSTEM/File.cpp#L497

We actually also have other fallbacks. We just thought it was a good idea to make it explicit. We could try to remove it and see if the relative default locations work.

Reply:
Please let's try the fallbacks. Setting and unsettling environment variables is error prone especially in nested environments.